### PR TITLE
Fix Pop Freq Cancer SV Variants View

### PR DESCRIPTION
**Pull request by ivadym**
This PR fixes a bug: 
- Empty Population Frequency column in the Cancer SV Variants view

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by ivadym** [comment]
Tested it in stage and it successfully shows the tooltip:

Screenshot 2024-01-18 at 17 42 18

However, I'm not sure if gnomAD freqs are being parsed (doesn't seem so). I will look into it tomorrow.

File: scout/server/blueprints/variants/templates/variants/cancer-sv-variants.html [hunk 1]
```diff
@@ -111,11 +111,7 @@
     <td class="col-2"><span style="white-space: nowrap;">{{ variant.position|human_longint|safe }}</span></td>
     <td class="col-2"><span style="white-space: nowrap;">{{ 'inf' if variant.end == 100000000000 else variant.end|human_longint|safe }}</span></td>
     <td class="col-2"><span style="white-space: nowrap;">{{ '-' if variant.length == 100000000000 else variant.length|human_longint|safe }}</span></td>
-    <td>
-      {% if variant.gnomad_frequency %}
-        {{ variant.gnomad_frequency|human_decimal }}
-      {% endif %}
-    </td>
+    <td class="text-end">{{ frequency_cell_general(variant) }}</td>
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/cancer-sv-variants.html:114]: 👍

**Review by dnil — APPROVED** [review]
The GnomAD SV collection is not super rich, but nice to reuse the macro and bring back the tilde! 😸

**Comment by ivadym** [comment]
> The GnomAD SV collection is not super rich, but nice to reuse the macro and bring back the tilde! 😸

Maybe we should report values from Swegen/1000G/... for `cancer_sv` and/or `cancer` when there's nothing in the GnomAD SV collection. What do you think? Right now, that column is full of `~`.

https://github.com/Clinical-Genomics/scout/blob/ac28c1f9ad139da51b60ddea3bb59765a93a95e0/scout/server/blueprints/variants/templates/variants/components.html#L316-L324

File: scout/server/blueprints/variants/templates/variants/cancer-sv-variants.html (hunk 1 above)
Comment (ivadym) [reply scout/server/blueprints/variants/templates/variants/cancer-sv-variants.html:114]: Thanks for the review!!

**Comment by dnil** [comment]
> > The GnomAD SV collection is not super rich, but nice to reuse the macro and bring back the tilde! 😸
> 
> Maybe we should show values from Swegen/1000G/... for `cancer_sv` and/or `cancer` when there's nothing in the GnomAD SV collection. What do you think? Right now, that column is full of `~`.
> 
> https://github.com/Clinical-Genomics/scout/blob/ac28c1f9ad139da51b60ddea3bb59765a93a95e0/scout/server/blueprints/variants/templates/variants/components.html#L316-L324

Good question in general; there is certainly more `~` than frequencies also for the RD side. I had hopes for the next releases of GnomAD SV, but they kind of ended up only on hg38. 🙄 I still can't see a good way to add multiple sources in the same column without making it either confusing or very big, but feel free to suggest something! Otherwise there is always the hover, which should be nicely populated, and the Local frequencies.

**Pull request merged**